Sprint-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+const dir = filePath.slice(0, filePath.lastIndexOf(base));
```

Comment:
Have you tried executing this program and output the value of the variables using `console.log()` to see if the code can produce the result you expect?

I don't think the value assigned `dir` is correct.

Reply:
Sorry. I overlooked. Code is correct